src/lib.rs

```diff
@@ -408,6 +445,44 @@ fn parse_wasm_features(features: &str) -> Result<wasmparser::WasmFeatures> {
         memory64: false,
     })
 }
+
+fn parse_wasi_modules(modules: &str) -> Result<WasiModules> {
+    let modules = modules.trim();
+    match modules {
+        "all" => Ok(WasiModules::default()),
+        "all-experimental" => Ok(WasiModules::all()),
+        "-all" | "-all-experimental" => Ok(WasiModules::none()),
```

Comment:
I think it's ok to avoid matching `-all-experimental` here since `-all` suffices for this use case.

Reply:
After discussing today, this now only matches `default` and `-default`.